### PR TITLE
feat(apps): Add Morpho app to zapper studio

### PR DESCRIPTION
**Pull request by julien-devatom**
## Description

Morpho is an on-chain peer-to-peer layer on top of lending pools. Rates are seamlessly improved for borrowers and lenders while preserving the same guarantees.

https://www.morpho.xyz

- add Morpho Compound contract positions fetcher
- add Morpho Compound balances fetcher

## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] (optional) As a contributor, my Ethereum address/ENS is:
- [ ] (optional) As a contributor, my Twitter handle is:

### CODE REVIEW

File: src/apps/morpho/morpho.definition.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const MORPHO_DEFINITION = appDefinition({
+  id: 'morpho',
+  name: 'Morpho',
+  description: 'TODO',
```
Comment (immasandwich) [inline src/apps/morpho/morpho.definition.ts:9]: Let's fill this in!

**Review by immasandwich — APPROVED** [review]
Nice PR sir. Just one comment and let's kick it in.

File: src/apps/morpho/morpho.definition.ts (hunk 1 above)
Comment (julien-devatom) [reply src/apps/morpho/morpho.definition.ts:9]: woops

Comment (julien-devatom) [reply src/apps/morpho/morpho.definition.ts]: done @immasandwich

**Comment by julien-devatom** [comment]
@immasandwich is it ok for you?

**Comment by julien-devatom** [comment]
@immasandwich It is ready to merge

**Pull request merged**